### PR TITLE
Modified Dockerfile to replace alpine and for slightly better efficiency

### PR DESCRIPTION
**Pull request by denizul**

### CODE REVIEW

**Comment by hchybz** [comment]
- works under Windows with docker desktop
 ==> had to change `dockerfile: Dockerfile`in docker-compose.yml

- does not work under WSL : same error as usual : can't go outside WSL network
```
(base) ogree@wOGrEE:/OGrEE/dockerfiletest/OGrEE-API$ docker-compose up
[+] Running 0/2
 ⠏ api_docs Pulling                                                                                                                                                                                                                                                                                    10.0s
[+] Running 0/2ing                                                                                                                                                                                                                                                                                     10.0s
 ⠋ api_docs Pulling                                                                                                                                                                                                                                                                                    10.0s
 ⠋ mongodb Pulling                                                                                                                                                                                                                                                                                     10.0s
error getting credentials - err: exit status 1, out: ``
```

**Comment by GeremWD** [comment]
I suspect even a
```
docker run hello-world
```
would not work, and in that case the problem would have nothing to do with ogree, but only with the docker setup.

File: Dockerfile [hunk 1]
```diff
@@ -1,49 +1,32 @@
...
+FROM busybox:latest
 USER root
-
-# Configure Go
-ENV GOROOT /usr/lib/go
-ENV GOPATH /go
-ENV PATH /go/bin:$PATH
 WORKDIR /home
 
 ADD . /home/
```
Comment (helderbetiol) [inline Dockerfile:32]: why not add the entrypoint directly in the dockerfile at the end?

Comment (GeremWD) [reply Dockerfile:32]: +1 for the entrypoint in the dockerfile rather than in the docker-compose

**Comment by GeremWD** [comment]
- It would be nice to remove one of the two dockerfile and one of the two .env.
- The .env file should not be copied on the container, as it is loaded by the docker-compose.
- ```
  RUN go install .
  ```
  seems like a simpler way to install the dependencies.

File: Dockerfile (hunk 1 above)
Comment (denizul) [reply Dockerfile:32]: I added the entrypoint. All good!

**Comment by GeremWD** [comment]
docker-compose was still pointing to dev.dockerfile so I fixed it, I believe everything is good now.

**Review by GeremWD — APPROVED** [review]

**Pull request merged**